iree/test/e2e/xla_ops/round.mlir

```diff
@@ -0,0 +1,7 @@
+func @tensor() attributes { iree.module.export } {
+  %input = iree.unfoldable_constant dense<[-0.7, -0.5, -0.2, 0.0, 0.2, 0.5, 0.7]> : tensor<7xf32>
+  %result = "mhlo.round_nearest_afz"(%input) : (tensor<7xf32>) -> tensor<7xf32>
```

Comment:
Are there other round ops to handle? I see `mhlo::RoundOp` in C++ but `mhlo.round_nearest_afz` here in .mlir. Is there something missing?

Reply:
So this actually is the mhlo::RoundOp ..... yay inconsistent naming? Sidenote - this does not match to TF's round op.